document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/data/accounts/AccountsManager.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.companieshouse.document.generator.accounts.data.accounts;
+
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.companieshouse.api.model.accounts.Accounts;
+import uk.gov.companieshouse.environment.impl.EnvironmentReaderImpl;
+
+public class AccountsManager {
+
+    /** represents the Authorization header name in the request */
+    private static final String AUTHORIZATION_HEADER = "Authorization";
+
+    private static final RestTemplate restTemplate = createRestTemplate();
```

Comment:
How are methods using the RestTemplate going to be tested if it's statically created on class load?

Reply:
This is the alternative `private-sdk` implementation. Currently we cannot JUnit test it. This is the same in the `company-accounts.api` also.